builds/misc/images-release-base-image-update.yaml

```diff
@@ -34,7 +34,7 @@ stages:
       - task: ShellScript@2
         displayName: "Build Azure-IoT-Edge-Core"
         inputs: 
-          args: "-c Release"
+          args: "-c $(Build.Configuration)"
```

Comment:
Since this is a release build, we should force the configuration to Release, instead of using a variable (that someone could accidentally set to Debug).

Reply:
The variable is not configurable from the pipeline. If we want to hardcode it, we have to hardcode it through and through.

Reply:
What I mean is, the original line here was correct: `args: "-c Release"`. There's no doubt what that argument is, and it will never change.